sdk/keyvault/keyvault-keys/src/lro/delete/operation.ts

```diff
@@ -0,0 +1,113 @@
+import { AbortSignalLike } from "@azure/abort-controller";
+import { PollOperationState, PollOperation } from "@azure/core-lro";
+import { RequestOptionsBase } from "@azure/core-http";
+import { DeletedKey, KeyClientInterface } from "../../keysModels";
+
+/**
+ * @interface
+ * An interface representing the state of a delete key's poll operation
+ */
+export interface DeleteKeyPollOperationState extends PollOperationState<DeletedKey> {
+  /**
+   * @member {string} [name] The name of the key that will be deleted
+   */
+  name: string;
+  /**
+   * @member {RequestOptionsBase} [requestOptions] The optional HTTP parameters that will be used to dele the key
+   */
+  requestOptions?: RequestOptionsBase;
+  /**
+   * @member {KeyClientInterface} [client] An instance of the key's client
+   */
+  client: KeyClientInterface;
+  /**
+   * @member {DeletedKey} [initialResponse] The initial response received the first time the service was reached by the operation's update function
+   */
+  initialResponse?: Error;
```

Comment:
this shouldn't be 'Error' right?

Reply:
I'll remove initialResponse and previous response